frontend/sw360-portlet/src/main/webapp/WEB-INF/customTags.tld

```diff
@@ -738,6 +738,17 @@
         </attribute>
     </tag>
     <tag>
+        <name>ComponentName</name>
```

Comment:
Looking at `customTags.tld`, it seems that almost all tags start with the prefix "Display". Maybe we should promote this to a strict convention for this project and call this tag `DisplayComponetName` (and accordingly for the other tags)?
Maybe the `out`-tag can be an exception as it is a general purpose thing....  ?

Reply:
Well in our customTags we have some extra tags to show only the Name of the object.
e.g. tags: RelaseName, ProjectName, LicenseName.
As a result I added this for the component.